### PR TITLE
Adding wrappers_pb.js to web packed files so that you can use nullable values

### PR DESCRIPTION
**Pull request by sknick**

### CODE REVIEW

**Comment by peuter** [comment]
But this always adds them no matter if you use them or not, right? I would prefer to have them optionally included by the user if he needs / uses them and only document how to do that.

File: README.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ module.exports {
   // do not end lines of generated code with ';' if true
   withoutSemi: false,
   // class to use for repeated properties
-  repeatedClass: 'qx.data.Array',
```
Comment (peuter) [inline README.md:139]: Also a change I am not ok with, this removes the opportunity to add change listeners to the property that are provided by qx.data.Array but not by the native Array. And that is the reason why qx.data.Array is used here in the first place.

File: config.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ const defaultConfig = {
   // do not end lines of generated code with ';' if true
   withoutSemi: false,
   // class to use for repeated properties
-  repeatedClass: 'qx.data.Array',
```
Comment (peuter) [inline config.js:35]: see above

**Review by peuter — CHANGES_REQUESTED** [review]

**Comment by peuter** [comment]
@sknick One general request: Please use meaningful commit messages. When I merge this is it is part of the projects commit history and no one will be able to see what the commits are all about.

File: README.md (hunk 1 above)
Comment (sknick) [reply README.md:139]: Ya, this PR is directly from my master branch so it's picking up changes I didn't intend to have in the PR. Let's just close this PR for now and I'll circle back around when I have things adjusted.

**Pull request closed without merging**